programs/witness_node/main.cpp

```diff
@@ -358,16 +362,17 @@ fc::optional<fc::logging_config> load_logging_config_from_ini_file(const fc::pat
             fc::path file_name = section_tree.get<std::string>("filename");
             if (file_name.is_relative())
                file_name = fc::absolute(config_ini_filename).parent_path() / file_name;
-            
+            int interval = section_tree.get_optional<int>("rotation_interval").get_value_or(60);
+            int limit = section_tree.get_optional<int>("rotation_limit").get_value_or(7);
```

Comment:
Please change default value to `1` (day) here for backward compatibility (avoid disk-full issues on existing nodes after upgraded). Keep the change (7 days) above for default value for new nodes.